Update redis.md (#2901)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-bindings/redis.md

An HTTP 204 (No Content) and empty body is returned if successful.

> - `get`

Reviewer: Let's model this section similarly to the [postgresql component](https://docs.dapr.io/reference/components-reference/supported-bindings/postgres/#binding-support)... so from lines 68 - 91, use the following formatting:

```
- `create`
- `get`
- `delete`

### create

You can store a record in Redis using the `create` operation. This sets a key to hold a value. If the key already exists, the value is overwritten.

#### Request

```json
{
  "operation": "create",
  "metadata": {
    "key": "key1"
  },
  "data": {
    "Hello": "World",
    "Lorem": "Ipsum"
  }
}
```
```
#### Response

An HTTP 204 (No Content) and empty body is returned if successful.

### get
```

You can get a record in Redis using the `get` operation. This gets a key that was previously set.

> ### Request

Reviewer: ```suggestion
#### Request
```

```json
{
  "operation": "get",
  "metadata": {
    "key": "key1"
  },
  "data": {
  }
}
```

> ### Response

Reviewer: ```suggestion
#### Response
```

```json
{
  "data": {
    "Hello": "World",
    "Lorem": "Ipsum"
  }
}
```

> - `delete`

Reviewer: ```suggestion
### delete
```

You can delete a record in Redis using the `delete` operation. Returns success whether the key exists or not.

> ### Request

Reviewer: ```suggestion
#### Request
```

```json
{
  "operation": "delete",
  "metadata": {
    "key": "key1"
  }
}
```

> ### Response

Reviewer: ```suggestion
#### Response
```

An HTTP 204 (No Content) and empty body is returned if successful.

> - `get`

Reviewer 2: @wxbty Add it to the list of operations above also. Also add `delete`

Author: The action list has got and delete added, what are you referring to?

> You can get a record in Redis using the `get` operation. This get a key where you set.

Reviewer 2: ```suggestion
You can get a record in Redis using the `get` operation. This gets a key that was previously set.
```

Author: ok

### Request

```json
{
  "operation": "get",
  "metadata": {
    "key": "key1"
  },
  "data": {
  }
}
```

### Response

```json
{
  "data": {
    "Hello": "World",
    "Lorem": "Ipsum"
  }
}
```

- `delete`

> You can delete multiple records in Redis using the `delete` operation. There is at least one key in metadata, {key: xxx}, if there are more than one, append key1, key2...

Reviewer 2: @wxbty 
in the code PR there is no option for deleting multiple keys at the same time ... I think this whole section on `delete` needs to be changed ... 
cc @ItalyPaleAle

Author: Because of @ItalyPaleAle's suggestion, the multi-key operation was removed. doc is not synchronized in time
